base-container/src/main/java/com/telecominfraproject/wlan/core/server/security/WebSecurityConfig.java

```diff
@@ -865,41 +855,42 @@ public boolean matches(HttpServletRequest request) {
      * 
      * @param providerIndex
      * @param defaultProperties
-     * @return null if clientId is not set.
+     * @return null if clientSecret is not set.
      * @throws Exception
      */
     protected Auth0AuthenticationProvider createAuth0AuthenticationProvider(int providerIndex) throws Exception {
-        String clientId;
         String clientSecret;
         String issuer;
         String accessTypeValue;
         String jwksLocation;
+        String claimsUrl;
         if (0 == providerIndex) {
-            clientId = environment.getProperty("tip.wlan.auth0.clientId", DEFAULT_AUTH0_PROPERTY);
             clientSecret = environment.getProperty("tip.wlan.auth0.clientSecret", DEFAULT_AUTH0_PROPERTY);
             issuer = environment.getProperty("tip.wlan.auth0.issuerUri", DEFAULT_AUTH0_PROPERTY);
             accessTypeValue = environment.getProperty("tip.wlan.auth0.accessType",
                     getDefaultAccessType(providerIndex));
             jwksLocation = environment.getProperty("tip.wlan.auth0.jwksLocation", DEFAULT_AUTH0_PROPERTY);
+            claimsUrl = environment.getProperty("tip.wlan.auth0.callbackUrl"); // See Auth0UserDetails
```

Comment:
why tip.wlan.auth0.callbackUrl here and tip.wlan.auth0.claimsUrl below?

Reply:
It should be callbackUrl. Fixed in latest commit
CallbackUrl was originally used for sending user authentication emails, but is no longer used. The custom namespace can be any url because it is only used for naming convention, but preferably a company owned domain, so the callbackUrl fits here since it would be pointing back to a company domain url.

Reply:
This raises more questions - why is the value for the claimsUrl is coming from the property tip.wlan.auth0.callbackUrl? In a couple of months when somebody tries to make sense of this it will be confusing. It would be better to maintain consistency in names. The system is complex enough already :)